helm/action/list.go

```diff
@@ -10,13 +10,15 @@ import (
 // List lists all of the local charts.
 func List(homedir string) {
 	md := filepath.Join(homedir, WorkspaceChartPath, "*")
+
 	charts, err := filepath.Glob(md)
-	if err != nil {
```

Comment:
@carmstrong is right that the Go idiom is always to do `varname != nil`. Go will catch the case of accidental assignment.